optuna/visualization/_pareto_front.py

```diff
@@ -220,7 +220,7 @@ def _get_pareto_front_info(
     if constraints_func is not None:
         feasible_trials = []
         infeasible_trials = []
-        for trial in study.get_trials(states=(TrialState.COMPLETE,)):
+        for trial in study.get_trials(deepcopy=False, states=(TrialState.COMPLETE,)):
```

Comment:
In my understanding, `trial` is not modified in this file, so we an use `deepcopy=False`.